tenets/codelingo/code-review-comments/avoid-renaming-imports/codelingo.yaml

```diff
@@ -1,11 +1,17 @@
 funcs:
-  - name: importNamesAreDifferent
+  - name: importNamesEqual
     type: asserter
     body: |
       function(a, b) {
         var aSplit = a.toLowerCase().split("/")
         var bSplit = b.toLowerCase().split("/")
-        return a[a.length - 1] !== b[b.length - 1]
+        return a[a.length - 1] === b[b.length - 1]
+      }
+  - name: neq
```

Comment:
It would be more natural to have this as eq and then negate it in the query if CLQL supports that.

eq would then make a good generic function to import.